frontend/src/router/index.ts

```diff
@@ -68,30 +67,13 @@ const router = createRouter({
       component: ContactView
     },
     {
-      path: '/data/:view/:dataset',
-      component: DataDashboardDatasetView,
-      meta: { reload: true },
-    },
-    {
-      path: '/data/:view',
-      redirect: '/data-dashboard'
-    },
-    {
-      path: '/data',
```

Comment:
Can we either keep the `/data` redirect and point to the new welcome view path, or edit our existing dashboard routing? I believe we have buttons on the header, homepage, and individual book views that break here.

Reply:
I'm not seeing any links to /data from any .vue file.

Reply:
[frontend/src/views/Header.vue](https://github.com/Spelman-College/spelman-dashboard/blob/main/frontend/src/views/Header.vue) line 15 uses `router.push` syntax to link to /data. I believe the homepage and book views use this approach as well.

Reply:
_fixed_